weaviate/collections/query.py

```diff
@@ -332,8 +332,8 @@ class _GroupByCollection(
 #     def near_video(
 #         self,
 #         near_video: str,
-#         certainty: Optional[float] = None,
-#         distance: Optional[float] = None,
+#         certainty: Optional[Union[float, int]] = None,
+#         distance: Optional[Union[float, int]] = None,
 #         auto_limit: Optional[int] = None,
 #         filters: Optional[_Filters] = None,
 #         return_metadata: Optional[MetadataQuery] = None,
```

Comment:
do we still need this commented out code?

Reply:
Haha, CTRL + F & replace! No, I think we can safely delete this come back to ORM afresh